### ISSUE TITLE
How to access child <Scene> props from parent's tabBarComponent?

### ISSUE THREAD

**Issue by rayj10** [aksonov/react-native-router-flux#3044]
### Version
Tell us which versions you are using: 

- react-native-router-flux v4.0.0-beta.28

Sorry for the noob question, but if I have the following structure:

```
<Scene tabs={true} tabBarComponent={()=><ScrollableTabBar items={["Approval", "My Request", " DO Customer", "My Confirm", "View Request"]}/>} tabBarPosition='top' lazy={true}>
       <Scene key="Home" hideNavBar component={Home} title={"Home"} name='Home'/>
</Scene>
```

How can I access child scene's (in this case the scene with `key='Home'`) `props` like `title`, `name`, or `key` from within `ScrollableTabBar`?

Thanks in advance for the pointers! :)

**Comment by daviscabral** [comment]
@rayj10 are not you able to access it from `this.props.title`?

**Comment by rayj10** [comment]
@daviscabral not from within ScrollableTabBar... besides it would be weird if say Home is not the only child of the scene with ScrollableTabBar wouldn't it? 

I have changed my structure anyhow because I found a better way of achieving what I wanted to. But just for future reference I would still like to know how can I access the array of children and their props from a component loaded onto parent scene i.e. ScrollableTabBar?

I know that usually it's available via `this.props.children` but thats if I have something like:

<ScrollableTabBar>
            <Child1>
            <Child2>

but if I have this:

<Scene tabBarComponent={ScrollableTabBar}>
            <Child1>
            <Child2>

`this.props.children` is returning undefined from within `ScrollableTabBar`
